### ISSUE TITLE
Add nullable annotation

### ISSUE THREAD

**Issue by maxkoshevoi** [LuckyDucko/Mopups#4]
Add `<Nullable>enable</Nullable>` to the project file.
This will provide better development and usage experience.

I can help annotate the code.

**Comment by LuckyDucko** [comment]
i agree, i always forget about these options, i'll go through and try and find a bunch of available extras we could switch on.

**Comment by maxkoshevoi** [comment]
Enabled them in general, but there're couple of warnings that need to be fixed. Most of them in `PopupPageRenderer.cs`

**Comment by LuckyDucko** [comment]
Hmmm, this already looks like a pain, but i guess when more features are implemented, we can see where we stand with these in `PopupPageRenderer`
